python_lib/qss_compiler/examples/parse.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Using this notebook requires manual install of MLIR python bindings:
```

Comment:
Are these comments about how to use the notebook still relevant?

Reply:
Same question for the notes in the example notebook.

Reply:
Removed in https://github.com/openqasm/qe-compiler/pull/264/commits/02bafaedd12067bd0eaffd7566ee37306f01e303